scripts/bundle

```diff
@@ -55,9 +59,12 @@ if (options.minify) {
       },
       mangle: {
         toplevel: true,
-        // https://github.com/mishoo/UglifyJS2#mangle-properties-options
         properties: {
-          regex: /^_/
+          regex: /^_/,
+          cache: {
+            cname: -1,
+            props: new Dictionary()
+          }
```

Comment:
This is pretty cool, and probably worth blogging about. You can share a dictionary of minified properties between builds. This will use the same minification dictionary across all modules for a build.